SA3/qa_web/static/css/base.css

```diff
@@ -1,69 +1,67 @@
-.good_answer{
-  color: green;
+.good-answer {
```

Comment:
CSS properties like this need to be one line. Only use this formatting for CSS properties taking up multiple lines.

Reply:
In the [Google HTML/CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#CSS), this is the preferred way of applying properties, even if they do not take mutliple lines.